9-regular-expressions/04-regexp-anchors/article.md

```diff
@@ -1,34 +1,34 @@
-# Anchors: string start ^ and end $
+# Âncoras: início ^ e fim $ de string
 
-The caret `pattern:^` and dollar `pattern:$` characters have special meaning in a regexp. They are called "anchors".
+Os caracterees acento circunflexo `pattern:^` e cifrão `pattern:$` possuem um significado especial em expressões regulares. Eles são chamados de "âncoras".
 
-The caret `pattern:^` matches at the beginning of the text, and the dollar `pattern:$` -- at the end.
+O acento circunflexo `pattern:^` corresponde ao início da string, e o cifrão `pattern:$` -- ao final.
 
-For instance, let's test if the text starts with `Mary`:
+Neste exemplo, vamos testar se o texto começa com `Mary`:
 
 ```js run
 let str1 = "Mary had a little lamb";
 alert( /^Mary/.test(str1) ); // true
 ```
 
-The pattern `pattern:^Mary` means: "string start and then Mary".
+O padrão `pattern:^Mary` quer dizer: "início da string, e então Mary"
 
-Similar to this, we can test if the string ends with `snow` using `pattern:snow$`:
+Da mesma maneira, podemos testar se a string termina com `snow` usando `pattern:snow$`:
 
 ```js run
 let str1 = "its fleece was white as snow";
 alert( /snow$/.test(str1) ); // true
 ```
 
-In these particular cases we could use string methods `startsWith/endsWith` instead. Regular expressions should be used for more complex tests.
+Nesses casos em particular, poderíamos usar os métodos do objeto string `startsWith/endsWith` em seu lugar. Expressões regulares devem ser usadas para testes mais complexos.
 
-## Testing for a full match
+## Testando por casamento completo
```

Comment:
I recommend just using the _Testando o casamento completo_ or _Testando a correspondência total_.

Reply:
@nazarepiedady I changed it completely to "Casando com uma string inteira" which better reflects the original intent of the section, even though its not a literal translation.